### ISSUE TITLE
train issue

### ISSUE THREAD

**Issue by hao-liu-china** [taotaoorange/SINT#1]
how to train?

**Comment by vincennnnt** [comment]
@mozhuangb sorry to interrupt u，i use the fast-rcnn-master，and get the hdf5 files，but how to train the code and load the hdf5 files？

**Comment by taotaoorange** [comment]
hdf5 files need to be specified in SINT_train.prototxt.
You need to make a solver.prototxt.  
##
base_lr: 0.001
lr_policy: "step"
gamma: 0.1
stepsize: 120000 #2ep
display: 50
max_iter: 600000 
momentum: 0.9
weight_decay: 0.001
##
Once you have the solver, training is just one command line.  Please also see http://caffe.berkeleyvision.org/gathered/examples/imagenet.html

**Comment by vincennnnt** [comment]
@taotaoorange Very appreciate to get your answer，i wrote the address of my hdf5 file to the train_hdf5.txt and began to train，but after the 130000 Iteration，the loss didn't descend，with staying fluctuate around 0.4，which was starting from the beginning.Hope to get your reply!  ^<>^

**Comment by lycqu** [comment]
@vincennnnt  I meet the same problem as you!the loss   stay around 0.4.have you  resolved the problem?And i do research in  Tracking.

**Comment by taotaoorange** [comment]
Did you check the validation loss? Does it go down?

**Comment by lycqu** [comment]
@taotaoorange  loss does not go down. loss  is  always  between  0.4~0.5.

**Comment by lycqu** [comment]
@taotaoorange  Can I communicate with  you  on Wechat?My Wechat ID  is :18827384395。

**Comment by taotaoorange** [comment]
In my experiment, the val loss went down in the first few epochs. The training was then stopped when the val loss did not go further down.

**Comment by lycqu** [comment]
@taotaoorange   I debug  the code .I found  that the label of data  are all zeros and  the 'd' is also zeros.I only delete the 'RNG'  func in the matlab code  and correct some dirs .

**Comment by lycqu** [comment]
@taotaoorange The loss go down  from 0.2 to 0.01~0.05 after 10000 Iteration.I found the loss is not right.
